src/ansi-c/expr2c.cpp

```diff
@@ -56,9 +56,9 @@ irep_idt expr2ct::id_shorthand(const irep_idt &identifier) const
 {
   const symbolt *symbol;
 
-  if(!ns.lookup(identifier, symbol) &&
-     !symbol->base_name.empty() &&
-      has_suffix(id2string(identifier), id2string(symbol->base_name)))
+  if(
+    ns && !ns->get().lookup(identifier, symbol) && !symbol->base_name.empty() &&
```

Comment:
Maybe pull `ns && ns->get().lookup` into a function so not every if has to be cluttered with an `if(ns)`